lisa/features/infiniband.py

```diff
@@ -442,15 +442,15 @@ def install_open_mpi(self) -> None:
         # Install Open MPI
         wget = node.tools[Wget]
         tar_file_path = wget.get(
-            "https://download.open-mpi.org/release/open-mpi/v4.0/openmpi-4.0.5.tar.gz",
+            "https://download.open-mpi.org/release/open-mpi/v4.1/openmpi-4.1.5.tar.gz",
             file_path=self.resource_disk_path,
             executable=True,
             overwrite=False,
             sudo=True,
         )
         tar = node.tools[Tar]
         tar.extract(tar_file_path, self.resource_disk_path, gzip=True, sudo=True)
-        openmpi_folder = node.get_pure_path(f"{self.resource_disk_path}/openmpi-4.0.5")
+        openmpi_folder = node.get_pure_path(f"{self.resource_disk_path}/openmpi-4.1.5")
```

Comment:
This version should be removed to reduce change cost next time.

Reply:
updated